app/views/users/show.html.erb

```diff
@@ -1,9 +1,11 @@
 <h1><%= @user.first_name %>'s Folio</h1>
 
+<%= link_to "Create New Folder", new_folder_path, class: "btn" %>
 <h3><%= @user.root_folder.name %></h3>
 
 <h3><% @user.folders.each do |folder| %></h3>
       <%= folder.name %>
+      <br>
     <% end %>
 <ul>
 <% @user.uploads.each do |upload| %>
```

Comment:
thinking of probably making this a table, based off of the wireframes @alfosco @JonathanSR @AELSchauer ?